docs/faq.md

```diff
@@ -53,6 +53,33 @@ However, for most participants, only a single sample was collected and submitted
 Because of this, many of the samples do not have a separate participant ID.
 Participant IDs are only present for samples that were derived from the same participant as at least one other sample.
 
+## What is a multiplexed sample? 
+
+Multiplexed samples refer to samples that have been combined together into a single library using cell hashing ([Stoeckius _et al._ 2018](https://doi.org/10.1186/s13059-018-1603-1)) or a related technology and then sequenced together. 
+This means that a single library contains cells or nuclei that correspond to multiple samples. 
+Each sample has been tagged with a hashtag oligo (HTO) prior to mixing, and that HTO can be used to identify which cells or nuclei belong to which sample within a multiplexed library. 
+The libraries available for download on the portal have not been separated by sample (i.e. demultiplexed), and therefore contain data from multiple samples.  
+
+## Why are demultiplexed samples not available? 
+
+Downloading a multiplexed sample on the portal will result in obtaining the gene expression files corresponding to the library containing the chosen multiplexed sample and any other samples that were multiplexed with that chosen sample. 
+This means that users will receive the gene expression data for all samples that were combined into a given library and will have to separate any cells corresponding to the sample of interest before proceeding with downstream analysis.
+
+We have applied multiple {ref}`demultiplexing methods <processing_information:hto demultiplexing>` to multiplexed libraries and noticed that these demultiplexing methods assign different cell barcodes to each sample at varying confidence levels. 
+Because of the inconsistency across demultiplexing methods used, the choice of demultiplexing method to use is up to the discretion of the user. 
+Rather than separating out each sample, the sample calls and any associated statistics regarding sample calls for multiple demultiplexing methods can be found in the `_filtered.rds` file for each multiplexed library. 
```

Comment:
I wanted to add a bit about the raw data here. We might want this sentence at the end though, not before the link to the demux results section?
```suggestion
Rather than separating out each sample, the sample calls and any associated statistics regarding sample calls for multiple demultiplexing methods can be found in the `_filtered.rds` file for each multiplexed library. 
We also include the Hash Tag Oligo counts matrix to allow demultiplexing using other available methods.
```